sale_exception/tests/test_sale_exception.py

```diff
@@ -51,18 +51,16 @@ def test_sale_order_exception(self):
         p = self.env.ref('product.product_product_7')
 
         # set ignore_exception = False  (Done by onchange of order_line)
-        self.assertRaises(
-            ValidationError,
-            so1.write,
-            {
-                'ignore_exception': False,
-                'order_line': [(0, 0, {'name': p.name,
-                                       'product_id': p.id,
-                                       'product_uom_qty': 2,
-                                       'product_uom': p.uom_id.id,
-                                       'price_unit': p.list_price})]
-            },
-        )
+        with self.assertRaises(ValidationError), self.env.cr.savepoint():
+            so1.write(
+                {
```

Comment:
Is this formatted with black, isn't it? Not very aesthetic and line.saving I have to say...

Reply:
No it isn't. Maybe in a near future :smile:

Reply:
Ah, then why putting the `{` in a new line instead on the previous, saving one line and, most important, an indentation level?

Reply:
I'm in a good mood. Done

Reply:
haha, thanks. It was genuine interest to know if there's a reason for that.